controllers/passenger.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import APIRouter, Depends
+
+from component_factory import get_passenger_service
+
+from model.requests.passenger import PassengerDriveOrderRequest, PassengerGetDrive
+from model.drive_order import DriveOrder
+from model.responses.passenger import DriveOrderResponse, GetDriveResponse
+from service.passenger_service import PassengerService
+
+router = APIRouter()
+
+
+def _to_drive_order_response(order: DriveOrder):
+    return DriveOrderResponse(order_id=order.id)
+
+
+def _to_get_drive_response(drive_id: int):
+    return GetDriveResponse(drive_id=drive_id)
+
+
+def _address_to_location(address: str):
+    # TODO convert address to array of two floats (lat, lon)
+    return [1, 1]
+
+
+async def add_drive_order(
```

Comment:
no routes assigned here

Reply:
see example in rating router

Reply:
routes are now in file be_routing - which should handle also passengers's requests.